spinnaker/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# CHANGELOG - Spinnaker
+
```

Comment:
```suggestion

<!-- towncrier release notes start -->

```

Could you add this to the md. We use towncrier to compile our changelogs. One of the validations will fail other wise.